### PR TITLE
Nesrine/vehicule

### PR DESCRIPTION
**Pull request by NESRINERABAAOUI**

### CODE REVIEW

File: client/src/components/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function Home() {
+  return <div style={{ backgroundColor: "purple", height: "84dvh" }}>HOME</div>;
```
Comment (lxndrMr) [inline client/src/components/Home.jsx]: You mean "84vh" ?

File: client/.prettierrc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "trailingComma": "es5",
+  "trailingComma": "all",
```
Comment (lxndrMr) [inline client/.prettierrc]: Restore the basic parameter

File: client/src/components/Info.jsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Info = () => {
+  return <div style={{ backgroundColor: "blue", height: "84dvh" }}>Info</div>;
```
Comment (lxndrMr) [inline client/src/components/Info.jsx]: You mean "84vh" ?

File: client/src/components/NavBar.jsx [hunk 4]
```diff
@@ -98,6 +98,12 @@ function NavBar() {
               Our destinations
             </Link>
           </li>
+          <li>
+            <Link to="/OurVehicles" aria-label="link to the Traffic info page">
+              {" "}
```
Comment (lxndrMr) [inline client/src/components/NavBar.jsx]: Remove comment

File: client/src/components/Voyage.jsx [hunk 5]
```diff
@@ -0,0 +1,5 @@
+function Voyage() {
+  return <div>Voyage</div>;
```
Comment (lxndrMr) [inline client/src/components/Voyage.jsx]: Why use div?

File: client/src/components/Vehicule.jsx [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+  const handleFavClicked = () => {};
+  return (
+    <main className="mainOurDestination">
+      <ReactPlayer
+        className="videoPlayer"
+        url="https://youtu.be/QGMxGdzYVY8"
+        width="100%"
+      />
+
+      <div className="cards">
```
Comment (lxndrMr) [inline client/src/components/Vehicule.jsx]: Why use div?

File: client/src/components/Vehicule.jsx [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+        {vehicules.map((vehicule) => (
+          <div key={vehicule.id} className="cardVehicule">
```
Comment (lxndrMr) [inline client/src/components/Vehicule.jsx]: Why use div?

File: client/src/components/Vehicule.jsx [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+            <img className="vehiculeImg" src={vehicule.img} alt="img" />
+            <h2 className="vehiculeName">{vehicule.nom} </h2>
+            <h4>Description: {vehicule.description} </h4>
+            <div className="carac">
```
Comment (lxndrMr) [inline client/src/components/Vehicule.jsx]: Why use div?

File: client/src/components/Vehicule.jsx [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+              <p>
+                Size <br /> {vehicule.Size}{" "}
+              </p>
+            </div>
+          </div>
+        ))}
+      </div>
+      <section className="generic">
+        <h1 className="vehiculeName">Equipments</h1>
+        <div className="divWidth">
```
Comment (lxndrMr) [inline client/src/components/Vehicule.jsx]: same

File: client/src/components/Vehicule.jsx [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+          <Slide>
+            {equipementsIMG.map((slideImage) => (
+              <div className="generic" key={slideImage.caption}>
```
Comment (lxndrMr) [inline client/src/components/Vehicule.jsx]: same

File: client/src/components/Info.jsx (hunk 3 above)
Comment (lxndrMr) [reply client/src/components/Info.jsx]: Why use div?

File: client/src/styles/Vehicules.scss [hunk 11]
```diff
@@ -0,0 +1,109 @@
+@use "./app.scss" as *;
+.cards {
+  display: flex;
+  width: 100dvw;
```
Comment (lxndrMr) [inline client/src/styles/Vehicules.scss:4]: You mean "100vw" ?

File: client/src/styles/Vehicules.scss [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+}
+.cardVehicule {
+  @each $property, $value in $glassmorphisme {
+    #{$property}: $value;
+  }
+  text-shadow: 0px 0px 5px gray;
+  padding: 4rem;
+  border: 2px solid white;
+  border-radius: 40px;
+  width: 30dvw;
```
Comment (lxndrMr) [inline client/src/styles/Vehicules.scss:15]: You mean "30vw" ?

File: client/src/styles/Vehicules.scss [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+.cardVehicule:hover {
+  background-color: rgb(18, 13, 18);
+  cursor: pointer;
+}
+
+.vehiculeName {
+  text-align: center;
+  color: aquamarine;
+  margin-bottom: 4dvh;
+  margin-top: 4dvh;
```
Comment (lxndrMr) [inline client/src/styles/Vehicules.scss:43]: You mean "4vh" ?

Comment (lxndrMr) [reply client/src/styles/Vehicules.scss:43]: i think "rem" it's better

File: client/src/styles/Vehicules.scss [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+.vehiculeImg {
+  width: 25rem;
+  height: 15 rem;
```
Comment (lxndrMr) [inline client/src/styles/Vehicules.scss]: no space between 15rem

File: client/src/styles/Vehicules.scss [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+  object-fit: cover;
+  border-radius: 40px;
+}
+
+.carac {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  margin-top: 3dvh;
```
Comment (lxndrMr) [inline client/src/styles/Vehicules.scss:58]: i think "rem" it's better

File: client/src/styles/Vehicules.scss [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+.spanStyle {
+  padding: 20px;
+  background-color: transparent;
+  color: #fff;
+  margin-top: 4dvh;
```
Comment (lxndrMr) [inline client/src/styles/Vehicules.scss:65]: same

File: client/src/styles/Vehicules.scss [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+    width: 80dvw;
+    height: 15 rem;
+    object-fit: cover;
+    border-radius: 40px;
+  }
+  .divWidth {
+    width: 80dvw;
+  }
+  .videoPlayer {
+    margin-top: 11dvh;
```
Comment (lxndrMr) [inline client/src/styles/Vehicules.scss:108]: i think "rem" it's better

File: client/src/styles/Vehicules.scss [hunk 18]
```diff
@@ -0,0 +1,109 @@
...
+    margin: 3rem;
+    color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+  }
+  .vehiculeImg {
+    width: 80dvw;
+    height: 15 rem;
```
Comment (lxndrMr) [inline client/src/styles/Vehicules.scss:100]: no space between 15rem

File: client/src/styles/Vehicules.scss [hunk 19]
```diff
@@ -0,0 +1,109 @@
...
+  color: #fff;
+  margin-top: 4dvh;
+}
+
+.divStyle {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-size: cover;
+  height: 30dvh;
```
Comment (lxndrMr) [inline client/src/styles/Vehicules.scss:73]: You mean "30vh" ?

File: client/src/styles/Vehicules.scss [hunk 20]
```diff
@@ -0,0 +1,109 @@
...
+
+.generic {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.divWidth {
+  width: 30dvw;
```
Comment (lxndrMr) [inline client/src/styles/Vehicules.scss]: You mean "30vw" ?

File: client/src/styles/Vehicules.scss [hunk 21]
```diff
@@ -0,0 +1,109 @@
...
+@media screen and (max-width: 992px) {
+  .cardVehicule {
+    @each $property, $value in $glassmorphisme {
+      #{$property}: $value;
+    }
+    text-shadow: 0px 0px 5px gray;
+    padding: 4rem;
+    border: 2px solid white;
+    border-radius: 40px;
+    width: 90dvw;
```
Comment (lxndrMr) [inline client/src/styles/Vehicules.scss:85]: You mean "90vw" ?

File: client/src/styles/Vehicules.scss [hunk 22]
```diff
@@ -0,0 +1,109 @@
...
+    margin: 3rem;
+    color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+  }
+  .vehiculeImg {
+    width: 80dvw;
```
Comment (lxndrMr) [inline client/src/styles/Vehicules.scss:99]: You mean "80vw" ?

**Review by lxndrMr — CHANGES_REQUESTED** [review]

File: client/.prettierrc (hunk 2 above)
Comment (ozen2) [reply client/.prettierrc]: yes

File: client/src/components/Home.jsx (hunk 1 above)
Comment (ozen2) [inline client/src/components/Home.jsx]: div ?????

File: client/src/components/NavBar.jsx (hunk 4 above)
Comment (ozen2) [inline client/src/components/NavBar.jsx]: why { " " } ?

File: client/src/components/Vehicule.jsx (hunk 6 above)
Comment (ozen2) [inline client/src/components/Vehicule.jsx]: div ????

File: client/src/components/Vehicule.jsx (hunk 7 above)
Comment (ozen2) [inline client/src/components/Vehicule.jsx]: div ????

File: client/src/components/Vehicule.jsx (hunk 8 above)
Comment (ozen2) [inline client/src/components/Vehicule.jsx]: div ???

File: client/src/components/Vehicule.jsx (hunk 9 above)
Comment (ozen2) [inline client/src/components/Vehicule.jsx]: div ?????

File: client/src/components/Vehicule.jsx (hunk 10 above)
Comment (ozen2) [inline client/src/components/Vehicule.jsx]: not div !!!!!

File: client/src/components/Voyage.jsx (hunk 5 above)
Comment (ozen2) [inline client/src/components/Voyage.jsx]: no div !!!!!!!!!!!

File: client/src/main.jsx [hunk 23]
```diff
@@ -12,6 +12,8 @@ import TrafficInfo from "./pages/TrafficInfo";
 import YourTrip from "./pages/YourTrip";
 import Contact from "./pages/Contact";
 import ErrorPage404 from "./pages/ErrorPage";
+import Vehicule from "./components/Vehicule";
+import vehicules from "./db/VehiculesDb";
```
Comment (ozen2) [inline client/src/main.jsx:16]: component double

File: client/src/styles/Vehicules.scss (hunk 11 above)
Comment (ozen2) [reply client/src/styles/Vehicules.scss:4]: yeah ??

File: client/src/styles/Vehicules.scss (hunk 12 above)
Comment (ozen2) [reply client/src/styles/Vehicules.scss:15]: same

File: client/src/styles/Vehicules.scss (hunk 13 above)
Comment (ozen2) [inline client/src/styles/Vehicules.scss:43]: we said we work with rem

File: client/src/styles/Vehicules.scss (hunk 14 above)
Comment (ozen2) [reply client/src/styles/Vehicules.scss:40]: yes

File: client/src/styles/Vehicules.scss (hunk 15 above)
Comment (ozen2) [reply client/src/styles/Vehicules.scss:50]: same

File: package-lock.json
Comment (ozen2) [inline package-lock.json:1]: why this file changed ??

File: package.json
Comment (ozen2) [inline package.json:1]: and this file ???

**Review by pjroggy — APPROVED** [review]

**Review by lxndrMr — APPROVED** [review]

File: client/src/components/Home.jsx
Comment (Yoann-Fortin) [inline client/src/components/Home.jsx:1]: This component is not update with rest code

File: client/src/components/NavBar.jsx [hunk 24]
```diff
@@ -98,6 +98,11 @@ function NavBar() {
               Our destinations
             </Link>
           </li>
+          <li>
+            <Link to="/OurVehicles" aria-label="link to the Traffic info page">
```
Comment (Yoann-Fortin) [inline client/src/components/NavBar.jsx]: attribut aria-label do not correspond with this real redirection

File: client/src/components/Voyage.jsx
Comment (Yoann-Fortin) [inline client/src/components/Voyage.jsx:1]: Not necessary  component

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**